channels/rdpear/test/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+
+set(MODULE_NAME "TestRdpear")
+set(MODULE_PREFIX "TEST_RDPEAR")
+
+set(TEST_RDPEAR_DRIVER TestRdpear.c)
+
+set(TEST_RDPEAR_COMMON_SRCS
+    ../ndr.c
```

Comment:
maybe create an object library from these as they are used in both, test and client?